### ISSUE TITLE
May 17th, 2024 is a holiday in Georgia

### ISSUE THREAD

**Issue by arkid15r** [vacanza/python-holidays#1795]
https://georgiatoday.ge/on-may-17-to-be-a-holiday-in-georgia/

**Comment by arkid15r** [comment]
Yeah, I couldn't find any confirmation it's a country wide holiday.

> The holiday applies primarily to employees of administrative bodies as defined by the General Administrative Code of Georgia.

https://1tv.ge/lang/en/news/may-17-to-be-day-off-in-georgia/

Also it's not that clear if it's one-time or annual holiday. Need to dig deeper into the GE gov sources.

**Comment by KJhellico** [comment]
The relevant decree is not yet available on [government website](https://www.gov.ge/index.php?lang_id=GEO&sec_id=602).

**Comment by arkid15r** [comment]
> The relevant decree is not yet available on [government website](https://www.gov.ge/index.php?lang_id=GEO&sec_id=602).

Let's wait for the official update? I'm not sure if we should rush it into the Monday's release.

**Comment by PPsyrius** [comment]
I found [this source](https://www.interpressnews.ge/ka/article/799688-mtavrobis-dadgenilebit-ojaxis-sicmindisa-da-mshoblebis-pativiscemis-dgestan-dakavshirebit-17-maisi-ukme-dged-gamocxadda/) in Georgian which got its wording - ended up cross-referencing them to [N167](https://www.facebook.com/story.php/?story_fbid=479231227780225&id=100070800565852)

> The relevant ordinance was published on the Legislative Herald on May 15 and is signed by the Prime Minister of Georgia.
>
> "In accordance with Article 30(3) of the Organic Law of Georgia "Labor Code of Georgia", May 17, 2024 shall be defined as a holiday for employees in administrative bodies ('administrative body') in connection with the day of family purity and respect for parents.
>
> An administrative body shall determine the services/employees under his/her system, with respect to whom this ordinance shall not apply, if the performance of their functions is an urgent necessity for the exercise of state authorities and the normal functioning of the society.
>
> This ordinance shall not apply to financial obligations to persons subject to the supervision of the National Bank of Georgia.
>
> This ordinance shall not apply to activities related to public procurement provided for by the Law of Georgia on Public Procurement, including the activities of the Council for Dispute Resolution related to procurement provided for by Article 3(1)(a.h) of this Law and related to public procurement", - reads the resolution.
